pkg/util/github/repo.go

```diff
@@ -23,11 +22,16 @@ func (c *Client) CreateRepo() error {
 }
 
 func (c *Client) Delete() error {
-	_, err := c.Client.Repositories.Delete(
+	response, err := c.Client.Repositories.Delete(
```

Comment:
Our code below uses `resp` to represent the `response`, maybe it can be consistent here, and use `resp` anywhere. Conversely, the following should also use the name `response`

Reply:
I actually prefer the full word here, and there are 3 reasons:

1. response is more widely used than resp (339M vs 62M according to GitHub)
2. Although in this context, "resp" isn't ambiguous, in other contexts, "resp" can mean other things, like respTabId -> responsive tab id. We should reduce ambiguity to make our code as easy to read as possible.
3. Compared to "response", "resp" isn't a lot shorter. Using the abbreviation doesn't save a whole lot of line width. If it's a tradition (like using "i" for loop counter) or can significantly reduce the line length (e.g., i18n vs internationalization, I think we can use the abbreviation)

So, I suggest in the future we can gradually replace "resp" to response.

Reply:
👍